<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Service">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Service</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GetPassword">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Get password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>220</y>
      <width>91</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add password</string>
    </property>
   </widget>
   <widget class="QLabel" name="Exist">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>File List.txt doesn't exiist</string>
    </property>
   </widget>
   <widget class="QLabel" name="Empty">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>File List.txt is empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="FieldSRV">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Field is empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="FieldPWD">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Field is empty</string>
    </property>
   </widget>
   <widget class="QLabel" name="Unable">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>191</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Unable to open the file List.txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="ExistSRV">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>This service does not exist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Quit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
